integration-tests/src/test/java/oracle/kubernetes/operator/ItPodsShutdownMS.java

```diff
@@ -687,7 +379,47 @@ public void testShutdownOptionsOverrideClusterLevel() throws Exception {
     LoggerHelper.getLocal().log(Level.INFO, "SUCCESS - {0}", testMethodName);
   }
 
-  private long verifyShutdown(long delayTime, Domain domain) throws Exception {
+  /**
+   * Call operator to scale to specified number of replicas.
+   *
+   * @param replicas - number of managed servers
+   * @throws Exception if fails to scale.
+   */
+  private void scaleCluster(int replicas, Domain domain) throws Exception {
+    LoggerHelper.getLocal().log(Level.INFO, "Scale up/down to " + replicas + " managed servers");
+    operator1.scale(domain.getDomainUid(), domain.getClusterName(), replicas);
+  }
+
+  /**
+   * Shutdown managed server and returns duration of shutdown in milliseconds.
+   *
+   * @throws Exception if failed to shutdown the server.
+   */
+  private static long shutdownServer(String serverName, String domainNS, String domainUid) throws Exception {
+    long startTime;
+    startTime = System.currentTimeMillis();
+    String cmd = "kubectl delete pod " + domainUid + "-" + serverName + " -n " + domainNS;
+    LoggerHelper.getLocal().log(Level.INFO, "command to shutdown server <" + serverName + "> is: " + cmd);
+    ExecResult result = ExecCommand.exec(cmd);
+    long terminationTime = 0;
+    if (result.exitValue() != 0) {
+      throw new Exception("FAILURE: command " + cmd + " failed, returned " + result.stderr());
+    }
+    TestUtils.checkPodCreated(domainUid + "-" + serverName, domainNS);
```

Comment:
Is this method is to only shutdown the server or check if server is recreated after delete ?

Reply:
switched to restartServer